projects/rero/ng-core/src/lib/record/editor/type/markdown/markdown.component.html

```diff
@@ -0,0 +1,22 @@
+<!--
+ RERO angular core
+ Copyright (C) 2021 RERO
+
+ This program is free software: you can redistribute it and/or modify
+ it under the terms of the GNU Affero General Public License as published by
+ the Free Software Foundation, version 3 of the License.
+
+ This program is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ GNU Affero General Public License for more details.
+
+ You should have received a copy of the GNU Affero General Public License
+ along with this program.  If not, see <http://www.gnu.org/licenses/>.
+-->
+<div class="my-2">
+  <textarea #textarea></textarea>
+  <div *ngIf="showError" class="invalid-feedback d-block">
+    <formly-validation-message [field]="field"></formly-validation-message>
+  </div>
+</div>
```

Comment:
HIGH COSMETIC : As this template is short and pretty "simple", It could be included into the component itself.